cmd/internal/apidump/apidump.go

```diff
@@ -80,6 +84,19 @@ var Cmd = &cobra.Command{
 			outFlag.AkitaURI = &uri
 		}
 
+		if postmanAPIKey != "" {
+			cfg.WritePostmanAPIKeyAndEnvironment("default", postmanAPIKey, postmanEnvironment)
+		}
```

Comment:
I don't think we should write the config file here, unfortunately.  That belongs in a login command if at all.

Reply:
I had done the changes based on the UI Mockup shared by Paul, [here](https://postman.slack.com/archives/C05EHCJ6KD5/p1689987756340769).
If we need to use `akita login` command then we should do the required changes in UI side also

Reply:
The point is that executing 'apidump' or 'inject' should not alter the user's configuration file.  I don't think we need to support 'login' -- though that would be fine if we did.  But the user should be able to provide an alternative ID in an environment variable without having that value persisted.